MozillaSocial/mozilla-social-ios
# Add accessibility checks for discover tab (MOS-14)

## Summary
Add accessibility checks for discover tab by running through the new accessibility audit test for iOS 17 and the Run Audit function in Xcode Accessibility Inspector.

## Implementation Details
* Add Previews to `RecommendationRow` to verify recommendation content layout in various font sizes
* Add `AccessibilityIdentiers` struct to define certain elements with an identifier and label to be used in Voice Over
* Account for navigation title for large sizes
* Also, had to address some SwiftLint checks that were giving errors from rebasing develop
* Added update to handle endpoint changes to accept image_sizes[] only

## Test Steps
* Run through the test in `MoSoContentUITests`
* Run through VoiceOver and Dynamic Type in the discover tab
* Confirm any accessibility issues

## PR Checklist:
